app/src/display/theme.c

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <lvgl.h>
+
+#ifdef CONFIG_ZMK_LV_FONT_DEFAULT_SMALL_MONTSERRAT_8
```

Comment:
Minor: use `#if defined()` for consistency with the following lines?

Reply:
Fixed.